README.md

```diff
@@ -5,6 +5,49 @@ This release, mirrored to GitHub, is the Dawn release, which contains the
 initial application, the MPC Powers-of-Tau Ceremony. Documentation for the
 underlying technology can be found at https://www.quilibrium.com/
 
+## Install Requirements
+
+    wget https://:go.dev/dl/go1.20.14.linux-amd64.tar.gz
+    sudo tar -xvf go1.20.14.linux-amd64.tar.gz
+    sudo mv go /usr/local
+    sudo rm go1.20.14.linux-amd64.tar.gz
+    sudo nano ~/.bashrc
+
+At the end of the file, add these lines and save the file.
+
+    GOROOT=/usr/local/go
+    GOPATH=$HOME/go
+    PATH=$GOPATH/bin:$GOROOT/bin:$PATH
+
+On command line, run 
+    ~/.bashrc
+
+Check GO Version
+    go version
+
+It must show "go version go.1.20.13 linux/amd64"
```

Comment:
These install requirements would install 1.20.14, so it should show `go version go1.20.14 linux/amd64`. That being said, the install instructions are specific to particular Linux distros, so we should probably have separate instructions for each platform if we're going to add these